### PR TITLE
track implementation and each launch

### PR DESCRIPTION
**Pull request by jaredmixpanel**
Add some additional developer experience internal tracking

### CODE REVIEW

File: Sources/Network.swift [hunk 1]
```diff
@@ -132,9 +132,9 @@ class Network {
                              properties: Properties = [:],
                              completion: ((Bool) -> Void)? = nil) {
         let trackProperties = properties.merging(["token": apiToken,
-                                                  "mp_lib": properties["mp_lib"] ?? "swift",
+                                                  "mp_lib": "swift",
                                                   "distinct_id": distinctId,
-                                                  "$lib_version": properties["$lib_version"] ?? AutomaticProperties.libVersion()]) {(current, _) in current }
+                                                  "$lib_version": AutomaticProperties.libVersion()]) {(current, _) in current }
```
Comment (jaredmixpanel) [inline Sources/Network.swift:137]: The `{(current, _) in current }` closure keeps the values in `properties` for any keys that are already present.

Comment (jaredmixpanel) [reply Sources/Network.swift:137]: https://developer.apple.com/documentation/swift/dictionary/3127175-merging

File: Sources/MixpanelInstance.swift [hunk 2]
```diff
@@ -152,19 +152,16 @@ open class MixpanelInstance: CustomDebugStringConvertible, FlushDelegate, AEDele
...
             } else {
                 Logger.info(message: "Logging Disabled")
                 Logger.disableLevel(.debug)
                 Logger.disableLevel(.info)
                 Logger.disableLevel(.warning)
                 Logger.disableLevel(.error)
+            }
 #if DEBUG
-                Network.sendHttpEvent(eventName: "Toggle SDK Logging", apiToken: "metrics-1", distinctId: apiToken, properties: ["Logging Enabled": false])
+            Network.sendHttpEvent(eventName: "Toggle SDK Logging", apiToken: "metrics-1", distinctId: apiToken, properties: ["Logging Enabled": true])
```
Comment (zihejia) [inline Sources/MixpanelInstance.swift:163]: "true" -> loggingEnabled

Comment (jaredmixpanel) [reply Sources/MixpanelInstance.swift:163]: good catch!

File: Sources/Network.swift [hunk 3]
```diff
@@ -173,15 +173,17 @@ class Network {
...
-        let engageData = JSONHandler.encodeAPIData([["$token": apiToken, "$distinct_id": distinctId, "$add": [eventName: 1]]])
-        if let engageData = engageData {
-            let engageBody = "data=\(engageData)".data(using: String.Encoding.utf8)
-            let engageResource = Network.buildResource(path: FlushType.people.rawValue,
-                                                      method: .post,
-                                                      requestBody: engageBody,
-                                                      headers: ["Accept-Encoding": "gzip"],
-                                                      parse: responseParser)
-            Network.apiRequest(base: BasePath.DefaultMixpanelAPI, resource: engageResource) { _, _, _ in } success: { _, _ in }
+        if !(eventName == "Integration") {
```
Comment (zihejia) [inline Sources/Network.swift:176]: better add a new param like `updatePeople`, it can be defaulted to true.

Comment (jaredmixpanel) [reply Sources/Network.swift]: done

**Review by zihejia — APPROVED** [review]

**Pull request merged**